Support alignment with beautify

FireFragment opened the pull request:
Beautify now outputs:
```
| left   | center | right |
| :----- | :----: | ----: |
| foo    |  bar   |   0.0 |
| barbar | foofoo |  10.0 |
```

instead of:

```
| left   | center | right |
| :----- | :----: | ----: |
| foo    | bar    | 0.0   |
| barbar | foofoo | 10.0  |
```

Closes #1 (and also [this issue from Abricotine](https://github.com/brrd/abricotine/issues/146))

## Add Table.getAlignment method

The PR also adds Table.getAlignment method, which returns alignment of any column. Alignment is determined by colons in the second line of the table.
The new beautify method uses this function, but it may be useful also in other scenarios.

FireFragment committed:
Add getAlignment method

Determines column alignment by colons at second line of table.

FireFragment committed:
Support alignment with beautify

Beautify method now aligns text in table by colons in second line.

brrd commented on src/celldown.coffee:
```diff
@@ -242,7 +254,14 @@ celldown = do () ->
-                        if missingChars > 0 then cell += (" " for [1..missingChars]).join("")
+                        # Used only if alignment isn't center
+                        spaces = if missingChars > 0 then (" " for [1..missingChars]).join("") else ""
+                        console.log @arr[rowIndex][colIndex]
```
Could you please remove this console.log?

FireFragment replied:
I'm sorry, I forgot to remove it.

brrd commented on src/celldown.coffee:
```diff
@@ -208,6 +208,18 @@ celldown = do () ->
             return this
 
+        # Get 'colIndex'th column alignment. Undefined means, that there isn't specified any align (no colons at the second line).
+        getAlignment: (colIndex) ->
```
Could you write the documentation about this new method in README.md?

brrd reviewed (changes requested):
Thank you very much for your contribution. It looks good, I only have two minor requests before merging. I'm available if you have any question.

FireFragment committed:
Remove unneeded console.log

In Table.beautify method.

FireFragment committed:
Documentation update

Update documentation with description of Table.getAlignment method and example of beautify following alignment.

FireFragment committed:
Table.getAlignment method now takes cursor position by default

If Table.getAlignment is called without arguments, it takes column of cursor

FireFragment committed:
Documentation update

Add informations about commit 4d0874.

FireFragment committed:
Merge branch 'beautify-align-support'

FireFragment commented:
I made changes you requested and also these two little tweaks:
 - `getAlignment` method now uses a column where is cursor when colIndex argument is not defined (like the `align` method)
 - Improved example of `beautify` method in README.md to show, how it aligns contents of the table.

brrd commented:
This looks good. Thank you for those changes.
